packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -542,35 +542,60 @@ class EditorPage {
 		return await blockLocator.text();
 	}
 
+	async getNumberOfParagraphBlocks() {
+		const paragraphBlockLocator = isAndroid()
+			? `//android.view.ViewGroup[contains(@content-desc, "Paragraph Block. Row")]/android.widget.EditText`
+			: `(//XCUIElementTypeButton[contains(@name, "Paragraph Block. Row")])`;
+		const locator = await this.driver.elementsByXPath(
+			paragraphBlockLocator
+		);
+		return locator.length;
+	}
+
+	async assertSlashInserterPresent() {
+		const slashInserterLocator = isAndroid()
+			? '//android.widget.HorizontalScrollView[@content-desc="Slash inserter results"]/android.view.ViewGroup'
+			: '(//XCUIElementTypeOther[@name="Slash inserter results"])[1]';
+
+		return await waitForVisible( this.driver, slashInserterLocator, true );
+	}
+
 	// =========================
 	// List Block functions
 	// =========================
 
-	async getTextViewForListBlock( block ) {
-		let textViewElementName = 'XCUIElementTypeTextView';
-		if ( isAndroid() ) {
-			textViewElementName = 'android.widget.EditText';
-		}
+	async getListBlock( options = { isEmptyBlock: false } ) {
```

Comment:
I noticed that this function doesn't provide an argument for querying a specific List block index, it would return always the first one, right?. Not sure if we have a case where we have multiple List blocks, but it would be great to have this functionality for future cases, wdyt?

Reply:
Yup you're right, currently, it doesn't look for the position and just returns the first one. Let me see if I can add the position to it.